test/src/sdk-harness/test_artifacts/tx_contract/src/main.sw

```diff
@@ -18,7 +19,8 @@ abi TxContractTest {
     fn get_tx_script_start_offset() -> u64;
 
     fn get_tx_input_pointer(index: u64) -> u32;
-    fn get_tx_input_type(ptr: u32) -> u8;
+    fn get_tx_input_type_from_ptr(ptr: u32) -> u8;
```

Comment:
I saw a PR from @AlicanC earlier today where all pointers were changed to `u64`. Should that be the same here ?

Reply:
I have no objections, consistency is good. Any obvious benefits to switching?

Reply:
+1